config_defaults.yml

```diff
@@ -41,6 +41,8 @@ appveyor.yml:
       RUBY_VER: 21-x64
   test_script:
   - "bundle exec rake spec SPEC_OPTS='--format documentation'"
+MAINTAINERS.md:
+  maintainers: "Puppet Forge Modules Team <forge-modules@puppet.com>"
```

Comment:
And that email isn't specific to us, are you planning on overriding it in each of the sync files?